.github/workflows/generate-pinval.yaml

```diff
@@ -187,6 +201,7 @@ jobs:
         with:
           role-to-assume: ${{ secrets.AWS_IAM_ROLE_TO_ASSUME_ARN }}
           aws-region: us-east-1
+          role-duration-seconds: 7200
```

Comment:
Now that we generate reports by neighborhood, builds for the largest towns can take slightly over an hour, which is the default timeout for IAM role authorization. This config bumps that timeout to two hours.